components/edit-collective/sections/Members.js

```diff
@@ -408,7 +363,7 @@ const memberFieldsFragment = gql`
   }
 `;
 
-const coreContributorsQuery = gql`
```

Comment:
I started a change in https://github.com/opencollective/opencollective-frontend/pull/6332/commits/aabfc85545ef61853949fba8c5955d3aa8b6c38d to avoid re-fetching the full list when editing entries. In such cases, fetching the `id` + the updated fields is enough for Apollo to update the cache manually.

However, this doesn't work currently because this query is still using GQLV1. We should instead use the GQLV2 equivalent.

Reply:
I will migrate it to GQLV2.